### PR TITLE
Update of pagination in popular movies

### PR DESCRIPTION
**Pull request by WoytekMig**
Solved the problem with pagination - now, the click on pagination starts loading immediately :)

### CODE REVIEW

**Review by Arcziwald — APPROVED** [review]
Great job :-)

File: src/features/MoviesBrowser/PopularMovies/index.js [hunk 1]
```diff
@@ -9,19 +9,32 @@ import MainHeader from "../../../common/MainHeader";
...
+    };
+  }, [currentPage]);
+
   return (
     <>
-      {moviesData.status === "loading" ? (
+      {isLoading ? (
         <Loading />
       ) : moviesData.status === "error" ? (
         <Error />
```
Comment (KrystianZybura) [inline src/features/MoviesBrowser/PopularMovies/index.js:40]: Wouldn't it be better to put it inside of the useMovieData custom hook just like you've done it with Error?

**Review by KrystianZybura — COMMENTED** [review]
Good job, works great, but the code can still be improved 💪

**Review by KrystianZybura — APPROVED** [review]

File: src/features/MoviesBrowser/PopularMovies/index.js (hunk 1 above)
Comment (WoytekMig) [reply src/features/MoviesBrowser/PopularMovies/index.js:40]: Agreed !

**Pull request merged**